assets/agenda/components/CoverageItemStatus.jsx

```diff
@@ -35,10 +36,11 @@ export default class CoverageItemStatus extends React.PureComponent {
     render() {
         const {coverage} = this.props;
 
-        if (coverage.coverage_status && !this.state.wire) {
+        if (coverage.workflow_status && !this.state.wire) {
             return [
                 <span key="label" className='coverage-item__text-label mr-1'>{gettext('Status')}:</span>,
-                <span key="value">{coverage.coverage_status}</span>,
+                <span key="value">{gettext('coverage {{ state }} ',
+                    {state: get(WORKFLOW_STATUS_TEXTS, coverage.workflow_status, 'planned')})}</span>,
```

Comment:
Here is another translation

Reply:
here it will translate `coverage {{ state }}` and replace state with workflow status, but that must be translated on its own

Reply:
I see, cool I'll do that